src/dissect/dissect.c

```diff
@@ -593,7 +619,16 @@ static int parse_argv(int argc, char *argv[]) {
                 if (optind != argc)
                         return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                                "Expected no argument.");
+                break;
+
+        case ACTION_VALIDATE:
+                if (optind + 1 != argc)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                               "Expected an image file path as only argument.");
 
+                arg_image = argv[optind];
```

Comment:
I guess this has to be `strdup()ed`, since `arg_image` has a static destructor? Or maybe use `parse_path_argument()` as above?